src/runtime_src/core/tools/xbmgmt2/OO_Config.cpp

```diff
@@ -146,7 +257,8 @@ OO_Config::OO_Config( const std::string &_longName, bool _isHidden)
     ("security", boost::program_options::value<decltype(m_security)>(&m_security), "<add description>")
     ("runtime_clk_scale", boost::program_options::value<decltype(m_clk_scale)>(&m_clk_scale), "<add description>")
     ("cs_threshold_power_override", boost::program_options::value<decltype(m_power_override)>(&m_power_override), "<add description>")
-    ("show", boost::program_options::bool_switch(&m_show), "<add description>")
+    ("cs_reset", boost::program_options::value<decltype(m_cs_reset)>(&m_cs_reset), "<add description>")
+    ("showx", boost::program_options::bool_switch(&m_show), "<add description>")
```

Comment:
showx typo

Reply:
Actually not a typo, but a work-around until command parse can handle -show without it being captured by --show-hidden.

Reply:
Aah, interesting :)